.changeset/loud-dolphins-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+Updated all elements docs page with new summaries, images, and correct links
```

Comment:
as far as the changeset is concerned (i.e. the npm package), the docs pages are not important, rather, the changes to the custom-elements.json file are what are important here.
```suggestion
`<rh-avatar>`:
`<rh-code-block>`: Updated custom elements manifest summaries
```